apps/tlon-mobile/package.json

```
@@ -67,7 +67,7 @@
     "@tloncorp/ui": "workspace:*",
     "@urbit/aura": "^2.0.1",
     "@urbit/http-api": "3.2.0-dev",
-    "@urbit/nockjs": "^1.5.0",
+    "@urbit/nockjs": "github:Fang-/nockjs#m/modern-mug",
```

This can be master branch now. Or the specific commit, might be better. (Until we push a new minor version, of course.)

```suggestion
    "@urbit/nockjs": "github:urbit/nockjs#8ed20e5",
```